app/controllers/bugs_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BugsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authenticate_control!
+
+  def index
+    @bugs = Bug.where(game: current_game).order(:created_at)
+  end
+
+  def new
+    @bug = Bug.new
+    @teams = Team.all
+  end
+
+  def create
+    targetId = bug_params['target'].to_i
+    beneficiaryId = bug_params['beneficiary'].to_i
+    new_params = bug_params.except('target', 'beneficiary')
+    @bug = Bug.new(new_params)
+    @bug.target = Team.find(targetId)
+    @bug.beneficiary = Team.find(beneficiaryId)
+    @bug.klass = Message
```

Comment:
The 4 bug types are: Tech, Comms, Operative, Alien Agent.

Alien Bug is deprecated because you can't bug an electronic message (yet).  Tech bug and Operative Bug would let you see technology sheet or operative mission results respectively.  I'm not really a fan of using .klass as a means to make this work though?  I'm not sure I see the benefit.